rwengine/src/render/ObjectRenderer.cpp

```diff
@@ -275,11 +275,12 @@ void ObjectRenderer::renderVehicle(VehicleObject* vehicle,
         auto& wi = vehicle->physVehicle->getWheelInfo(w);
         // Construct our own matrix so we can use the local transform
         vehicle->physVehicle->updateWheelTransform(w, false);
+        bool isRhino = (vehicle->getVehicle()->vehiclename_ == "RHINO");
 
         auto up = -wi.m_wheelDirectionCS;
         auto right = wi.m_wheelAxleCS;
         auto fwd = up.cross(right);
-        btQuaternion steerQ(up, wi.m_steering);
+        btQuaternion steerQ(up, (isRhino) ? 0 : wi.m_steering);
```

Comment:
`typedef float btScalar`
I can make it either `(btScalar)0.f` or `0.f` instead -- whatever you like more